src/bots/reddit/utils.py

```diff
@@ -11,7 +11,7 @@ def parse_user(user: Redditors):
   i['comment_karma'] = user.comment_karma
   i['link_karma'] = user.link_karma
   i['username'] = user.name
-  i['created_utc'] = user.created_utc
+  i['created_utc'] = time.strftime('%Y-%m-%d %H:%M:%S', time.localtime(user.created_utc))
```

Comment:
I think  it would be better  to add a new field. called `created_utc_human` or  something in addition to the epoch. because it's easier to do date math with the epoch value.

Reply:
done.